logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/e2ei/usecase/FetchMLSVerificationStatusUseCase.kt

```diff
@@ -65,62 +62,57 @@ data class VerificationStatusData(
 )
 
 @Suppress("LongParameterList")
-internal class MLSConversationsVerificationStatusesHandlerImpl(
+internal class FetchMLSVerificationStatusUseCaseImpl(
     private val conversationRepository: ConversationRepository,
     private val persistMessage: PersistMessageUseCase,
     private val mlsClientProvider: MLSClientProvider,
     private val mlsConversationRepository: MLSConversationRepository,
-    private val epochChangesObserver: EpochChangesObserver,
     private val selfUserId: UserId,
     private val userRepository: UserRepository,
     kaliumLogger: KaliumLogger
-) : MLSConversationsVerificationStatusesHandler {
-
-    private val logger = kaliumLogger.withTextTag("MLSConversationsVerificationStatusesHandler")
-
-    override suspend fun invoke() {
-        logger.d("Starting to monitor")
-        epochChangesObserver.observe()
-            .collect { groupId ->
-
-                mlsClientProvider.getMLSClient()
-                    .flatMap { mlsClient ->
-                        wrapMLSRequest { mlsClient.isGroupVerified(groupId.value) }.map {
-                            it.toModel()
-                        }
-                    }.flatMap { ccGroupStatus ->
-                        if (ccGroupStatus == VerificationStatus.VERIFIED) {
-                            verifyUsersStatus(groupId)
-                        } else {
-                            conversationRepository.getConversationDetailsByMLSGroupId(groupId).map {
-                                VerificationStatusData(
-                                    conversationId = it.conversation.id,
-                                    currentPersistedStatus = it.conversation.mlsVerificationStatus,
-                                    newStatus =
-                                    ccGroupStatus
-                                )
-                            }
-                        }
-                    }.onSuccess {
-                        updateStatusAndNotifyUserIfNeeded(it)
+) : FetchMLSVerificationStatusUseCase {
+
+    private val logger = kaliumLogger.withTextTag("FetchMLSVerificationStatusUseCaseImpl")
```

Comment:
question: do we use this logger anywhere?

Reply:
yes, line 147, 167, 204 :)

Reply:
Doesn't look like its being used in this file anymore 🤔